acord/models/guild.py

```diff
@@ -56,7 +64,7 @@ class Guild(pydantic.BaseModel, Hashable):
 
     emojis: List[Emoji]
     """ List of emojis in guild """
-    explicit_content_filter: int
+    explicit_content_filter: ExplicitContentFilterLevel
```

Comment:
:class:\`GuildMessageNotification\`

Reply:
Done in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c